pages/portfolio.html

```diff
@@ -4,9 +4,51 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>✨Portfolio✨</title>
+  <link rel="stylesheet" href="../styles/style.css">
 </head>
+  <header>
+    <h1>K. Buslach </h1>
+    <h1>ADA C17 Sharks 🦈</h1>
+    <ul>
+      <nav>
```

Comment:
Nice, here I see the nav element is inside the header element. 

I would suggest your unordered list be nested inside your nav element. 

Then your header element should also go in the body element. 

```HTML
<body>
    <header>
        <nav>
            <ul>
                <li>
                <li>
                <li>